content/en/tracing/advanced/adding_metadata_to_spans.md

```diff
@@ -16,26 +16,18 @@ further_reading:
   text: "Explore your services, resources, and traces"
 ---
 
-Adding metadata in the form of key-value pairs to a span allows to correlate traces with other Datadog products to provide more details about specific spans. [Read more about tagging][1]
+Adding metadata in the form of key-value pairs to a span allows to correlate traces with other Datadog products to provide more details about specific spans. Metadata can be either added [to a single span](#adding-metadata-to-a-span) or [globally to all spans](#adding-metadata-globally-to-all-spans)
```

Comment:
I'd mention the difference here between Datadog `tags` and Tracing `tags`. 

Something along the lines that Datadog has the idea of `tags`, and point to the definition. And then say that tracing has the idea of `tags` too, but in this document and around the product, we differentiate between the two by calling them `metadata` when we mean the Span `tags`.